### PR TITLE
GEOS-6435 - Add INSPIRE extended capabilities to WCS

### PR DESCRIPTION
**Pull request by bgsmase**
…ilities response

### CODE REVIEW

**Comment by bgsmase** [comment]
https://osgeo-org.atlassian.net/browse/GEOS-6435

File: src/extension/inspire/src/main/java/org/geoserver/inspire/wcs/WCSExtendedCapabilitiesProvider.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
+/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
```
Comment (aaime) [inline src/extension/inspire/src/main/java/org/geoserver/inspire/wcs/WCSExtendedCapabilitiesProvider.java]: New file, the copyright is just "(c) 2015 - Open Source Geospatial Foundation - all rights reserved"

Comment (aaime) [inline src/extension/inspire/src/main/java/org/geoserver/inspire/wcs/WCSExtendedCapabilitiesProvider.java]: Oh... unless you copy/pasted it from another pre-existing provider... in that case the header would be right as is

Comment (aaime) [inline src/extension/inspire/src/main/java/org/geoserver/inspire/wcs/WCSExtendedCapabilitiesProvider.java]: Yeah, verified that is the case

**Comment by aaime** [comment]
Ok, all good, merging. Cheers for a well done and easy to review pull request ;-)

**Pull request merged**